lib/config.js

```diff
@@ -8,33 +8,27 @@ const GLOBAL_CONFIG = Symbol('globalConfig');
 const PROJECT_CONFIG = Symbol('projectConfig');
 const LOCAL_CONFIG = Symbol('localConfig');
 
-exports.GLOBAL_CONFIG = GLOBAL_CONFIG;
-exports.PROJECT_CONFIG = PROJECT_CONFIG;
-exports.LOCAL_CONFIG = LOCAL_CONFIG;
-
 function getNcurcPath() {
-  if (process.env.XDG_CONFIG_HOME !== 'undefined' &&
-      process.env.XDG_CONFIG_HOME !== undefined) {
-    return path.join(process.env.XDG_CONFIG_HOME, 'ncurc');
-  } else {
-    return path.join(os.homedir(), '.ncurc');
+  let configHome = os.homedir();
+  if (process.platform === 'linux') {
+    configHome = process.env.XDG_CONFIG_HOME || `${os.homedir()}/config`;
```

Comment:
I think it should be `${os.homedir()}/.config`

Reply:
whoops yes you're right! also need to figure out how to update the one failing test accordingly, since it's not really testing for the right signal anymore 🤔

Reply:
think that should do it